modules/claims_api/app/workers/claims_api/claim_establisher.rb

```diff
@@ -72,17 +72,17 @@ def queue_special_issues_updater(auth_headers, special_issues_per_disability, au
           code: disability['code'],
           name: disability['name']
         }
-        ClaimsApi::SpecialIssueUpdater.perform_async(bgs_user(auth_headers),
+        ClaimsApi::SpecialIssueUpdater.perform_async(auto_claim.evss_id,
```

Comment:
remove `auto_claim.evss_id`. the special issue updater only accepts 3 arguments as well, we provide 4 here.